### PR TITLE
added more swahili translation

### PR DESCRIPTION
**Pull request by rosteve**
# Description

Added more translation for Child schedule messages. also fixed some trailing space that was causing sentinel to crash in https://github.com/medic/medic-projects/issues/4500
cc  @abbyad @jillrshah

### CODE REVIEW

**Comment by abbyad** [comment]
Thanks @rosteve for updating the translations! Are you able to remove the translations which are the same as in the [3.0.x branch](https://github.com/medic/medic-webapp/blob/3.0.x/ddocs/medic/_attachments/translations/messages-sw.properties) and recommit? Once that is done we can merge.

**Comment by rosteve** [comment]
Hey @abbyad ,Sure, I have now removed the translations which are the same as in the [3.0.x](https://github.com/medic/medic-webapp/blob/3.0.x/ddocs/medic/_attachments/translations/messages-sw.properties) branch in the above commit.
The first three translation  were existing but I have updated them since these are the ones which had a trailing space and which had caused sentinel to break in https://github.com/medic/medic-projects/issues/4500

**Comment by abbyad** [comment]
Hi @rosteve, I have reviewed it again. Here are some notes:

- `report.delivery.delivery_code_label` and `report.delivery.pregnancy_outcome_label` seem to have accidentally become `report.delivery.label_delivery_code` and `report.delivery.label_pregnancy_outcome`
- `targets.*` and `schedule.child_health` are completely missing and are needed. If you aren't translating them yet leave them blank so that we know to come back to them.

Once these are complete we can import into POEditor to make sure they are included in subsequent releases of medic-webapp.

**Comment by garethbowen** [comment]
@rosteve Can you finish off the last changes requested by Marc? I'd love to get this merged.

**Comment by garethbowen** [comment]
Thanks @rosteve !

@abbyad Can you re-review please?

File: config/standard/translations/messages-sw.properties [hunk 1]
```diff
@@ -1,18 +1,74 @@
...
-messages.schedule.postnatal.day_3_overdue = Je, {{patient_name}} ({{patient_id}}) alihudhuria siku-3 ya ziara ya baada ya kujifungua? Tujulishe kwa kutuma 'M {{patient_id}}'. Asante!
-messages.schedule.postnatal.day_7 = Tafadhali ona kwamba  {{patient_name}} ({{patient_id}}) amehudhuria siku-7 ya ziara ya baada ya kuzaa kesho. Wakihudhuria, tujulishe kwa 'M {{patient_id}}'. Shukrani!
-messages.schedule.postnatal.day_7_overdue = Je, {{patient_name}} ({{patient_id}}) alihudhuria siku-7 ya ziara ya baada ya kujifungua? Tujulishe kwa kutuma 'M {{patient_id}}'. Asante!
-messages.schedule.postnatal.week_6 = Tafadhali ona kwamba {{patient_name}} ({{patient_id}}) amehudhuria wiki yao ya 6 ya ziara ya kujifungua. Wakihudhuria ziara iyo, tujulishe kwa 'M {{patient_id}}'. Shukrani!
-messages.schedule.postnatal.week_6_overdue = Je, {{patient_name}} ({{patient_id}}) alihudhuria ziara ya baada ya kujifungua ya wiki  ya 6? Tujulishe kwa kutuma   'M {{patient_id}}'. Shukrani!
-
-report.delivery.delivery_code_label = Kodi ya Kujifungua
+# Do not translate this label
```
Comment (abbyad) [inline config/standard/translations/messages-sw.properties]: Is there a reason to keep this?

```suggestion
```

**Comment by rosteve** [comment]
Hey @abbyad , just to be sure, should I add back the translation which I had removed [here](https://github.com/medic/medic-webapp/pull/4765#issuecomment-410427060) and then recommit?

**Comment by abbyad** [comment]
> should I add back the translation which I had removed here and then recommit?

No need to add any translations that are in 3.0.x, since those are now included directly in the webapp.

**Comment by abbyad** [comment]
The ones that are still blank in this PR are actually in the webapp in English, but not yet translated [there](https://github.com/medic/medic-webapp/blob/345066c88286d200767742d799aa6c1f32a032e2/ddocs/medic/_attachments/translations/messages-sw.properties#L1030-L1050), nor in [POEditor for Swahili](https://poeditor.com/projects/po_edit?order=&filter=&id=33025&id_language=154&search=targets.).  That means that they still need to be translated in the Standard translations AND POEditor. 

Please add those and I will re-review. Thanks!

**Comment by rosteve** [comment]
Hey @abbyad , I've now added the missing translations that were blank in this PR in the above commit and in Poeditor too

**Comment by garethbowen** [comment]
Closing as this is completely replaced by the new PR: https://github.com/medic/medic/pull/5367

**Pull request closed without merging**